VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -6,10 +6,8 @@ import { ReactTestInstance } from 'react-test-renderer'
 import { context, render, waitFor } from 'testUtils'
 import { PickerItem } from './VAModalPicker'
 import VAModalPicker from './VAModalPicker'
-import TextView from 'components/TextView'
-import BaseListItem from 'components/BaseListItem'
 import { InitialState } from 'store/slices'
-import { RenderAPI } from '@testing-library/react-native'
+import { RenderAPI, fireEvent, screen } from '@testing-library/react-native'
```

Comment:
move this import to the top with the other react native imports, also remove the import on the first line

Reply:
Updated.